packages/core/core-flows/src/order/workflows/claim-add-new-item.ts

```diff
@@ -0,0 +1,109 @@
+import {
```

Comment:
We need to replace all occurrences of `orderExchange` with orderClaim` in this file. I stopped highlighting them after I found a few of them